### PR TITLE
updating endpoints file

### PR DESCRIPTION
**Pull request by adriane-cardozo**
Updating endpoints-support.md file.

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -475,6 +494,35 @@
...
+|     **Switch Types**                                                                                                                              |
+|<sub>/rest/switch-types</sub>                                                            | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switch-types/{id}</sub>                                                       | GET      | :white_check_mark: | :white_check_mark: |    |
+|     **Switches**                                                                                                                                   |
+|<sub>/rest/switches</sub>                                                                | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switches/{id}</sub>                                                           | GET      | :white_check_mark: | :white_check_mark: |    |
+|<sub>/rest/switches/{id}</sub>                                                           | PATCH    | :heavy_minus_sign: | :black_square_button: |    |
```
Comment (CintiaLopes) [inline endpoints-support.md]: Actually this endpoint it is supported in Java, even though no method was added to the sample.

File: endpoints-support.md [hunk 2]
```diff
@@ -492,6 +540,8 @@
 |<sub>/rest/users/{userName}</sub>                                                        | DELETE   | :black_square_button: | :black_square_button: |    |
 |<sub>/rest/users/{userName}/roles</sub>                                                  | POST     | :black_square_button: | :black_square_button: |    |
 |<sub>/rest/users/{userName}/roles</sub>                                                  | PUT      | :black_square_button: | :black_square_button: |    |
+|     **Version**                                                                                                                         |
+|<sub>/rest/version</sub>                                                                 | GET      | :black_square_button: | :black_square_button: |                      |
```
Comment (CintiaLopes) [inline endpoints-support.md]: Actually this endpoint it is supported in Java, even though no sample was created. 
It is being used in OneViewConnector.

File: endpoints-support.md [hunk 3]
```diff
@@ -190,27 +198,30 @@
...
-|<sub>/rest/licenses</sub>                                                                |POST      |    | :white_check_mark: | :white_check_mark: |
-|<sub>/rest/licenses/{id}</sub>                                                           |GET       |    | :white_check_mark: | :white_check_mark: |
-|<sub>/rest/licenses/{licenseId}</sub>                                                    |GET       |    | :white_check_mark: | :white_check_mark: |
+|<sub>/rest/licenses</sub>                                                                |GET       | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+|<sub>/rest/licenses</sub>                                                                |POST      | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+|<sub>/rest/licenses/{id}</sub>                                                           |GET       | :white_check_mark: | :white_check_mark: | :white_check_mark: |
+|<sub>/rest/licenses/{licenseId}</sub>                                                    |GET       | :white_check_mark: | :white_check_mark: | :white_check_mark: |
```
Comment (CintiaLopes) [inline endpoints-support.md]: This one should be a DELETE.

**Review by CintiaLopes — COMMENTED** [review]
Also:
- URI: `/rest/server-hardware/{id}/javaRemoteConsole` should be updated to `/rest/server-hardware/{id}/javaRemoteConsoleUrl`
- `/rest/storage-volumes/repair` *(GET)* is not supported in Java.

File: endpoints-support.md (hunk 3 above)
Comment (adriane-cardozo) [reply endpoints-support.md]: Fixed

File: endpoints-support.md (hunk 1 above)
Comment (adriane-cardozo) [reply endpoints-support.md]: Updated

File: endpoints-support.md (hunk 2 above)
Comment (adriane-cardozo) [reply endpoints-support.md]: Updated

**Review by CintiaLopes — APPROVED** [review]

**Review by ianrosadas — APPROVED** [review]

**Pull request merged**